src/test/kotlin/org/jetbrains/exposed/sql/tests/DatabaseTestsBase.kt

```diff
@@ -85,7 +89,13 @@ abstract class DatabaseTestsBase {
 
         val database = Database.connect(dbSettings.connection, user = dbSettings.user, password = dbSettings.pass, driver = dbSettings.driver)
 
-        transaction(database.metadata.defaultTransactionIsolation, 1) {
+        val transactionIsolation = if (dbSettings.dialect == SQLServerDialect) {
```

Comment:
Is it really needed? In debug session I see that database.metadata.defaultTransactionIsolation returns Connection.TRANSACTION_READ_COMMITTED.

Reply:
I tried to revert and got `com.microsoft.sqlserver.jdbc.SQLServerException: The connection is closed.` Running on Mac.

Reply:
Sounds strange, because [official MS documentation](https://docs.microsoft.com/en-us/sql/t-sql/statements/set-transaction-isolation-level-transact-sql) states that READ COMMITTED is a default level. Do you run it against SQL Server ran in docker?

Reply:
```
public int getDefaultTransactionIsolation() throws SQLServerException {
        checkClosed();
        return java.sql.Connection.TRANSACTION_READ_COMMITTED;
    }
```
This is code from `com.microsoft.sqlserver.jdbc.SQLServerDatabaseMetaData` and it fails at `checkClosed();`. But I agree that this is weird code.